content/docs/tech-insights/displaying-fact-data/index.md

```diff
@@ -0,0 +1,71 @@
+---
+title: Displaying Fact Data
+publishedDate: '2024-08-12T14:33:22.0Z'
+description: Roadie Plugins to display fact data directly
+---
+
+## Introduction
+
+Tech Insights in Roadie provides the capabilities to automatically retrieve and store relevant data for your entities. With this data you can enhance the Tech Insights visibility within your Roadie instance and easily keep track, visualize and display important information for your entities. 
+
+## Components
+
+Roadie provides various different components to display fact data. These range from overall fact gathering table views to more fine-grained fact info displaying widgets. The use cases for the former larger components are to provide and overview of retrieved fact data within the system, allowing users to keep themselves up to date with the latest information at a glance. The more fine-grained, entity level, fact displaying widgets can be used to display relevant current and historical information about collected facts. 
```

Comment:
This paragraph is very terse and complicated. Can we split the sentences e.g.

The first sentence could be this:
"Roadie provides the ability to display various views of fact data from tech insights."

I don't understand this sentence: "These range from overall fact gathering table views to more fine-grained fact info displaying widgets."

Actually I think I might misunderstand most of the paragraph.